Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -456,7 +456,10 @@ private void ThreadExecution(CancellationToken cancellationToken)
                     }
                 }
 
-                EnsureSingleInstance();
+                if (_data.Value.IsOnline != null && _data.Value.IsOnline.Value)
```

Comment:
Please introduce here a comment explaining the cause of this `if` statement.

Reply:
Replacing "if" with catching the UnauthorizedAccessException exception